mods/ra2/rules/soviet-structures.yaml

```diff
@@ -32,6 +32,21 @@ nacnst:
 		Facing: 96
 		DeployCursor: undeploy
 		RequiresCondition: factundeploy && !build-incomplete
+	TransformsIntoMobile:
+		RequiresCondition: factundeploy
+		Locomotor: tracked
+		Voice: Move
+		RequiresForceMove: true
+	TransformsIntoRepairable:
+		RequiresCondition: factundeploy
+		RepairActors: nadept, caoutp, gadept
```

Comment:
`gadept, nadept, caoutp` please. Same applies allied one.

Reply:
It is exactly the same just in a different order.

Reply:
Yes, for consistency. I'm sure other places (Repairable in defaults) use that order.